sonic-sensormond/scripts/sensormond

```diff
@@ -0,0 +1,561 @@
+#!/usr/bin/python3
+
+"""
+    sensormond
+    Sensor monitor daemon for SONiC
+"""
+
+import signal
+import sys
+import threading
+import time
+from datetime import datetime
+
+import sonic_platform
+from sonic_py_common import daemon_base, logger
+from sonic_py_common.task_base import ProcessTaskBase
+from swsscommon import swsscommon
+
+
+SYSLOG_IDENTIFIER = 'sensormond'
+NOT_AVAILABLE     = 'N/A'
+CHASSIS_INFO_KEY  = 'chassis 1'
+INVALID_SLOT      = -1
+
+PHYSICAL_ENTITY_INFO_TABLE = 'PHYSICAL_ENTITY_INFO'
+
+# Python2 support.
```

Comment:
Why is python2 support needed? This should never be included in a release that launches it with anything other than python3

Reply:
I believe python2 support is not deprecated. Need to check for details but most common code seems to support python2.

Reply:
Checked with Msft. I will remove python2 support.